### DISCUSSION TITLE
Adding libraries?

### DISCUSSION THREAD

**Discussion by byoyomi** [MerrimanInd/drawpyo#45] [Q&A] [answered]
I need to use shapes from the "basic" library, which doesn't seem to be supported. Is it pretty straightforward for me to add this?

**Comment by MerrimanInd** [comment, marked as answer]
Yeah I think that should be straightforward. The documentation has some explanation of how it works [at this page](https://merrimanind.github.io/drawpyo/usage/shape_libs/). Taking a quick look at the styles for some of the shapes in that library I don't see any attributes that would cause issues. Just have to do the legwork copying it into a TOML file.

You can also use the apply_style_string() function documented [on this page](https://merrimanind.github.io/drawpyo/usage/objects/) if it's easier. Just copy the style string from the basic objects you want and you can apply that string directly to an object.
